### ISSUE TITLE
 Harmonization Page Has a Few TABLE Related Issues

### ISSUE THREAD

**Issue by jfc3-dol** [w3c/wai-website#427]
While reading the "Why Standards Harmonization is Essential to Web Accessibility" page ( https://www.w3.org/WAI/standards-guidelines/harmonization/ )  I noticed the TABLE was missing `scope="col"` for the TABLE headings (TH) along with the `SUMMARY `and `CAPTION`.

I would expect the TABLE headings (TH) to have `scope="col"` on them, and they seem to be missing. Also, I would expect a `CAPTION` and `SUMMARY `for the TABLE, but they could be deprecated.

Please see the attached screenshot with the highlighted code.

<img width="960" alt="harmonization page - table issues" src="https://user-images.githubusercontent.com/10423487/170285598-d3214df7-73ae-4ec1-adb7-fe522725c6b5.png">
